server/platforms/generic_csv.py

```diff
@@ -40,7 +40,7 @@ def _load_from_file(self, path_to_file: str) -> List[Dict]:
         for row in data:
             # this validates the date the same way to back-end parses it
             row['publish_date'] = dateutil.parser.parse(row['publish_date'])
-        self._data = data
+        return data
```

Comment:
I'm not able to "comment" on it, but check out line 28. I think this possibly fixes a bug where `_data` might be set to `None`.